packages/fiori/cypress/specs/SideNavigation.cy.tsx

```diff
@@ -116,6 +116,28 @@ describe("Side Navigation Rendering", () => {
 		cy.get("#item11").should("have.attr", "tooltip", TOOLTIP_TEXT);
 		cy.get("#item21").should("not.have.attr", "tooltip");
 	});
+
+	it("Tests disabled parent item", () => {
+		cy.mount(
+			<SideNavigation id="sn1">
+				<SideNavigationItem id="parent" expanded={true} text="Group 1">
+					<SideNavigationSubItem text="child 1" />
+					<SideNavigationSubItem disabled={true} text="child 2" />
+				</SideNavigationItem>
+			</SideNavigation>);
+
+		cy.get("#parent").should("not.have.attr", "disabled");
+		cy.get("#parent").invoke("prop", "disabled", true);
+		cy.get("#parent").should("have.attr", "disabled");
```

Comment:
This line is redundant. UI5 element already guarantees that setting a property will also update the attribute